HamzaGo5911/automated-web-crawler
# Added model layer for web crawler

This **PR** introduces the model layer for the automated web crawler project. The model layer defines the data structure used to represent entities that the crawler extracts information about. It includes a primary model, **EntityProfile**, which encompasses various fields such as name, contact information, images, occupation, family details, and political party affiliation.

How to Test:
Run the unit tests using **go test** to verify the correctness of the model and its methods.
